client/templates/minutes/minutesEditParticipants.js

```diff
@@ -84,6 +84,16 @@ Template.minutesEditParticipants.helpers({
         }
     },
 
+    switch2MultiColumn() {
+        let aMin = new Minutes(_minutesID);
+
+        if (!Session.get("global.isMobileWidth")) {
```

Comment:
I know we are using the `global.isMobileWidth` session variable a lot in our code. However, this is really not a good practice. Instead I would prefer using [css media rules](https://www.w3schools.com/cssref/css3_pr_mediaquery.asp). So simply wrap the css-class `multicolumn` inside the following block:
```
@media (min-width: 768px) {
    .multicolumn {
        -webkit-column-count: 2;
        -moz-column-count: 2;
        column-count: 2;
    }
}
```
This makes sure that the defined style definition is only available for devices with a min-width of 768px. Then you can remove the `switch2MultiColumn`-helper completely and set the css class statically in the template.